lib/rules/unit-blacklist/README.md

```diff
@@ -94,3 +94,46 @@ a { -moz-border-radius-topright: 40px; }
 ```css
 a { height: 100vmin; }
 ```
+
+### `ignoreMediaFeatureNames: { unit: ["property", "/regex/"] }`
+
+Ignore units for specific feature names.
+
+For example, with `["px", "dpi"]`.
+
+Given:
+
+```js
+{
+  "px": [ "min-width", "/height$/" ],
+  "dpi": [ "resolution" ]  
+}
+```
+
+The following patterns are *not* considered violations:
+
+```css
+@media (min-width: 960px) { }
```

Comment:
Can you please change the examples to use empty braces i.e. `{}` as per [the guidelines](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#example-patterns).

Reply:
Yep, sorry about that. Done in my last [commit](6a0bc46).